.github/workflows/ci.yml

```diff
@@ -41,8 +41,9 @@ jobs:
 
       - name: Install ⚙️
         run: |
-          python -m pip install --upgrade pip setuptools wheel
-          pip install --upgrade . -r dev-requirements.txt
+          python -m pip install --upgrade pip setuptools wheel pre-commit
```

Comment:
`requirements-dev.txt` already includes `pre-commit`
But that's a detail, I'm OK to merge this PR without changing that